tests/test_behav.py

```diff
@@ -641,3 +645,22 @@ def test_action_value_with_chance_player_action(game: gbt.Game, rational_flag: b
     chance_action = game.players.chance.infosets[0].actions[0]
     with pytest.raises(ValueError):
         game.mixed_behavior_profile(rational=rational_flag).action_value(chance_action)
+
+
+@pytest.mark.parametrize(
```

Comment:
This only tests with data that has the correct "shape".  There are no tests to ensure that bad input data is handled appropriate (with an error).